project2/src/main/java/com/revature/moneyRequest/model/MoneyRequest.java

```diff
@@ -0,0 +1,102 @@
+package com.revature.MoneyRequest.model;
```

Comment:
Same here. Shouldn't this be `.moneyRequest.model`?

Interesting, does Java not care about capitalizations within imports?

Reply:
i think they super do lemme fix these

Reply:
just checked and apparently java actually doesn't care! wow, TIL lol